js/tsconfig.test.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
is this extra file necessary?

Reply:
The short answer is yes. It's needed in order for ts-jest to work.

The long answer is that there's ways to play around with `tsconfig.json` that may allow for the removal everywhere of the `rootDir` prop, but it's almost certainly not worth the trouble. In order to ensure integration with various build tools from jlab core, we should keep `tsconfig.json` to the "standard" jlab setup